api/src/main/java/org/openmrs/module/appframework/service/AppFrameworkServiceImpl.java

```diff
@@ -348,11 +345,16 @@ public boolean checkRequireExpression(Requireable candidate, AppContextModel con
 					}
 				}
 
-				javascriptEngine.setBindings(bindings, ScriptContext.ENGINE_SCOPE);
-				ObjectMapper jackson = new ObjectMapper();
+				ObjectMapper objectMapper = new ObjectMapper();
+				for (Map.Entry<String, Object> e : contextModel.entrySet()) {
+					if (!(e.getValue() instanceof Map) && !"util".equals(e.getKey())) {
```

Comment:
Think about this again, what do you think of combining it with the other loop above on line 340?